HealthGPT/HealthGPT/HealthDataInterpreter.swift

```diff
@@ -0,0 +1,51 @@
+//
+// This source file is part of the Stanford HealthGPT project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University & Project Contributors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Spezi
+import SpeziFHIR
+import SpeziOpenAI
+import Foundation
+
+class HealthDataInterpreter<ComponentStandard: Standard>: DefaultInitializable, Component, ObservableObject, ObservableObjectProvider {
```

Comment:
Great job for creating a Spezi component and starting to move the code out of the view! 🚀